src/toil/job.py

```diff
@@ -1384,12 +1385,16 @@ def __init__(self, service):
         self.jobWrapper = None
 
     def run(self, fileStore):
+        
+        # we need access to the filestore from underneath the service job
+        self.fileStore = fileStore
```

Comment:
I think it should be pretty easy to make sure that every job has `.fileStore` but if you don't want to do this as part of this PR, could you create a ticket?